gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/impl/ApiDuplicatorServiceImplTest.java

```diff
@@ -15,14 +15,34 @@
  */
 package io.gravitee.rest.api.service.impl;
```

Comment:
Rename this class to `ApiDuplicatorService_RecalculateApiDefinitionIdsTest` and put it out of the `impl` package
Or
Move all others tests classes in `impl` package 😉

Reply:
As discussed together, the aim is to unit test internal methods without setting them public.
So we made them protected, and the Test class has to be in the same package as the source one in order to call it.

So, I renamed this file and moved all services tests to the impl package, as they all test implementations.